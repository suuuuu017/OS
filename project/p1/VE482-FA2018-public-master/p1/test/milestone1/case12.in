WRITELN echo 123 > 1.txt
WRITELN head < 1.txt
WRITELN diff -y - 1.txt < 1.txt
WRITELN ls
WRITELN exit
