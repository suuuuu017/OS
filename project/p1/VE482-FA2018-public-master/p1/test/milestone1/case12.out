mumsh $ mumsh $ 123
mumsh $ 123								123
mumsh $ 1.txt
driver
mumsh
mumsh_memory_check
mumsh $ exit
